spikemetrics/metrics.py

```diff
@@ -973,8 +977,6 @@ def find_neighboring_channels(peak_channel, channel_list, num_channels_to_compar
     # compute pairwise distance
     distances = [np.linalg.norm(peak_channel_location - loc) for loc in channel_locations]
     # get k closest channels (+1 because distance 0 is peak_channel)
-    neighboring_channels_inds = np.argsort(distances)[:num_channels_to_compare + 1]
+    neighboring_channels_inds = np.argsort(distances)[:num_channels_to_compare]
```

Comment:
Why did you re-include the peak channel in the neighboring channels?

Reply:
I did this because later what is returned by this function (`neighboring_channels`) is used simply as the channels over which to compute the metrics (e.g. when channels_to_use is defined), which should include the peak channel as well - sorry for the confusing terminology, I thought maybe it's OK to think of a channel as its own neighbor. But feel free to change this back to include only the true neighbors - in that case we would need to change some of the lines in the main for loop. Also feel free to delete the lines I commented out - I just kept them in case you wanted to revert...